### ISSUE TITLE
Quick drop in loss after only a few batches

### ISSUE THREAD

**Issue by joshuachang2311** [dmlc/gluon-cv#1396]
I played with with [4. Train FCN on Pascal VOC Dataset](https://gluon-cv.mxnet.io/build/examples_segmentation/train_fcn.html) and set the learning rate of lr_scheduler to 0.000000000001, but doesn't seems to stop the problem from happening. Training loss almost reached 0 after 2 epochs.
